docs/components/modeler/bpmn/link-events/link-events.md

```diff
@@ -0,0 +1,45 @@
+---
+id: link-events
+title: "Link events"
+description: "Link events are pairs of intermediate events, one throwing and one catching, that link the two together."
+---
+
+Link events are pairs of intermediate events, one throwing and one catching, that link the two together.
+
+Link events have no significance related to content, but facilitate the diagram-creation process and are a mechanism for connecting two sections of a process. Link events can be used as intermediate events only.
+
+:::tip
+You can use link events to create loops, to skip sections of a process, or to simplify the sequence flow lines in the diagram.
+:::
+
+Link event pairs have a throwing link event as the "exit point", and a catching link event as the "re-entrance point". The two events are marked as a pair by their link name.
+
+In practice, two paired link events function the same as two [intermediate none events] connected via a sequence flow.
+
+![A pair of link events is equivalent to a pair of intermediate none events connected via a sequence flow](./assets/link-events-example.png)
+
+Link events can be very useful if you draw comprehensive process diagrams with many sequence flows.
+
+Links help avoid what otherwise might look like a “spaghetti” diagram.
```

Comment:
What do you mean by a "spaghetti diagram" here? If a user is unfamiliar, I might just say something brief like "Links help avoid what otherwise might look like a “spaghetti” diagram, where XYZ occurs."